extensions/phylum.ts

```diff
@@ -37,19 +54,7 @@ export class PhylumApi {
     // Ensure header object is initialized.
     const fetchInit = init ?? {};
 
-    // Ensure consistent headers type.
-    fetchInit.headers = new Headers(fetchInit.headers);
-
-    // Set Authorization header if it is missing.
-    if (!fetchInit.headers.has("Authorization")) {
-      const token = await PhylumApi.getAccessToken();
-      fetchInit.headers.set("Authorization", `Bearer ${token}`);
-    }
-
-    // Set Content-Type header if it is missing.
-    if (fetchInit.headers.has("Content-Type")) {
-      fetchInit.headers.set("Content-Type", "application/json");
-    }
```

Comment:
If has, then set.... Whoops. We missed a `!` here.

Reply:
Good catch!